openhands/runtime/__init__.py

```diff
@@ -30,6 +28,6 @@ def get_runtime_cls(name: str):
     'RemoteRuntime',
     'ModalRuntime',
     'RunloopRuntime',
-    'EventStreamRuntime',
+    'DockerRuntime',
```

Comment:
@openhands-agent I think we need to import LocalRuntime in openhands/runtime/__init__.py